### PR TITLE
Create blockchain.md

### PR DESCRIPTION
**Pull request by yachee-gupta**
Added blockchain documentation

### CODE REVIEW

**Comment by yachee-gupta** [comment]
issue number #29

File: block-chain/blockchain.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+For example, a smart contract can be written to stipulate the cost of shipping an item where the shipping charge changes depending on how quickly the item arrives. Both the parties agrees to the terms written to the ledger, the appropriate funds change hands automatically when the item is received.
+![Smart_contract image](https://hyperledger-fabric.readthedocs.io/en/release-2.2/_images/Smart_Contract.png)
+
+Consensus is the process of keeping the ledger transactions synchronized across the network. It ensures that ledgers update only when transactions are approved by the appropriate participants, and that when ledgers do update, they update with the same transactions in the same order.
+
+**So, the blockchain is a shared, replicated transaction system which is updated via smart contracts and kept consistently synchronized through a collaborative process called consensus.**
+
+Reference- Bitcoin and Cryptocurrency Technologies book- https://bitcoinbook.cs.princeton.edu
```
Comment (animenon) [inline block-chain/blockchain.md]: Change this to the format: `[ref name](link)`

**Review by animenon — COMMENTED** [review]
Looks good, just one comment.

Also would be nice if you could highlight important points so that the doc is easier to follow.

**Comment by animenon** [comment]
Also would be nice if you could update exiting doc: [README.md](https://github.com/techfaqs/Tech-FAQs/blob/master/block-chain/README.md)
(as that page already has similar content)

File: block-chain/blockchain.md (hunk 1 above)
Comment (yachee-gupta) [reply block-chain/blockchain.md]: Done now.
Pl. accept PR #71

**Comment by yachee-gupta** [comment]
> Also would be nice if you could update exiting doc: [README.md](https://github.com/techfaqs/Tech-FAQs/blob/master/block-chain/README.md)
> (as that page already has similar content)

Updated the readme doc with the blockchain doc [Readme.md](https://github.com/yachee-gupta/Tech-FAQs/blob/patch-1/block-chain/README.md)

**Review by animenon — APPROVED** [review]

**Pull request merged**